api/desecapi/tests/test_crypto.py

```diff
@@ -0,0 +1,56 @@
+from math import log
+
+from django.test import TestCase
+
+from desecapi import crypto
+
+
+class CryptoTestCase(TestCase):
+    context = 'desecapi.tests.test_crypto'
+
+    def test_retrieved_key_is_reproducible(self):
+        keys = (crypto.retrieve_key(label='test', context=self.context) for _ in range(2))
+        self.assertEqual(*keys)
+
+    def test_retrieved_key_depends_on_secret(self):
+        keys = []
+        for secret in ['abcdefgh', 'hgfedcba']:
```

Comment:
Is the real secret you use longer than this? 😄  HKDF probably shouldn't be used for values this small.

Reply:
It is provided to the Docker container as an environment variable, so this code does not really care about it. But yes, our secret is much longer! :)